styles.css

```diff
@@ -0,0 +1,374 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body{
+   font-family: 'Times New Roman', Times, serif;
+   font-size: 100%;
+    height: 100%;
+}
+
+.head-container{
+    position:relative; 
+    width: 100%;
+    height: 45rem;
+    border: solid;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* background:linear-gradient(transperant, black, 30) url("./Assets/dying-light-2.jfif");
+    background-repeat: no-repeat;
+    background-size: cover;
+    filter: blur(); */
+}
```

Comment:
Hey @fahd23  Since the changes will occurs in main styles.css, hence i'll update them once they are merged to development branch or else if might raise issues

Reply:
Thank you soo much for you suggestion i'll apply them merged